packages/test-utils/src/create-dom-event.js

```diff
@@ -24,6 +24,19 @@ const modifiers = {
   pagedown: 34
 }
 
+function getOptions(modifier, { bubbles, cancelable }, options) {
+  const keyCode = modifiers[modifier] || options.keyCode || options.code
```

Comment:
`code` and `keyCode` are already spread in, if provided, here we are just ensuring we dont unnecessarily overwrite then with modifiers, right?

Reply:
Yes. Previously the bug was that modifiers was falsey, yet still overwriting the keyCode.